test/utils/expected-files.js

```diff
@@ -375,10 +375,10 @@ const expectedFiles = {
 
     client: [
         'angular.json',
+        'ngsw-config.json',
         '.eslintrc.json',
         'package.json',
-        'postcss.config.js',
```

Comment:
We need to delete every removed file in:
https://github.com/jhipster/generator-jhipster/blob/355fb16ea2d5feb08408083f1aaddfdb13bc461c/generators/cleanup.js#L107-L110

Reply:
Yes, I'll remove also the webpack files !